Assets/Scripts/UI/XboxRemap.cs

```diff
@@ -11,16 +11,18 @@ public class XboxRemap : MonoBehaviour
     int index;
     public string keyName;
     bool remaping;
-    XboxController xbox;
 
     public void Update()
     {
         if (remaping)
         {
-            if (xbox.AnyButtonDown)
+            if (InputManager.xboxControllers[0].AnyButtonDown)
```

Comment:
please use .firstOrDefault on collections when retrieving the first item

Reply:
done